src/application/machines.ts

```diff
@@ -87,12 +87,13 @@ export function createDevtoolsMachine({ actions }: { actions: Actions }) {
             timeout: "timedout",
             clientNotFound: "notFound",
           },
-          entry: ["unsubscribeFromAll", "connectToClient"],
+          entry: ["unsubscribeFromAll"],
```

Comment:
This ended up working very strangely because sometimes the `connectToClient` event was sent before a page was fully unloaded, which caused some false-positives on determining when the client reconnected. Instead, we now just rely on the content script sending the `registerContent` event to determine when to navigate back to the `connected` state

Reply:
```suggestion
          entry: "unsubscribeFromAll",
```